doc/all.mk

```diff
@@ -45,6 +45,7 @@ CONF_FILES := $(filter-out %~,$(wildcard raddb/*conf raddb/mods-available/* radd
 BASE_ADOC_FILES := $(wildcard doc/*/*.adoc) $(wildcard doc/*/*/*.adoc)
 AUTO_ADOC_FILES := $(patsubst raddb/%,doc/raddb/%.adoc,$(CONF_FILES))
 AUTO_ADOC_FILES += $(patsubst raddb/%.md,doc/raddb/%.adoc,$(shell find raddb -name "*\.md" -print))
+AUTO_ADOC_FILES += $(patsubst doc/%.md,doc/%.adoc,$(shell find doc -name "*\.md" -print))
```

Comment:
This is probably better off as `$(wildcard doc/*/*/*.md)`, like the adoc entry a few lines above